core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -77,6 +77,7 @@ object ApiVersion {
     // and KafkaStorageException for fetch requests.
     "1.1-IV0" -> KAFKA_1_1_IV0,
     "1.1" -> KAFKA_1_1_IV0,
+    // Introduced OffsetsForLeaderEpochRequest/OffsetsForLeaderEpochResponse V1 via KIP-279
```

Comment:
nits: to be more consistent with the existing comment, we can just say `Introduced OffsetsForLeaderEpochRequest V1 via KIP-279`